segmentstore/storage/impl/src/test/java/io/pravega/segmentstore/storage/impl/bookkeeper/RegularBookKeeperLogTests.java

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Let's put these two classes nested under `BookKeeperLogTests` since they aren't that verbose.

Reply:
`junit` does not like this.

Reply:
I have used this pattern a lot in numerous instances. Check out, as an example: `io.pravega.common.util.SortedIndexTestBase`

Reply:
Fixed.